runtime/dolphin/src/lib.rs

```diff
@@ -707,11 +728,14 @@ construct_runtime!(
         TechnicalMembership: pallet_membership::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 18,
 
         // Collator support. the order of these 5 are important and shall not change.
+        // Session must be included *before* AuthorInherent
```

Comment:
can you write explicitly why this is the case, please

Reply:
I could if it was the case 🤡 
See comment here for details https://github.com/Manta-Network/Manta/pull/582#discussion_r938256087

i've removed this comment